src/main/java/com/carservice/manager/service/VehicleService.java

```diff
@@ -18,25 +22,47 @@ public VehicleService(VehicleRepository vehicleRepository) {
     }
 
 
-    public void addVehicle(VehicleModel vehicleModel) {
+    public void addVehicle(VehicleModel vehicleModel, String fileName) {
         vehicleModel.setId(UUID.randomUUID().hashCode());
-        vehicleRepository.addVehicle(vehicleModel);
+        vehicleModel.setAdmissionDate(new Date());
+        vehicleRepository.addVehicle(vehicleModel, fileName);
     }
 
-    public List<VehicleModel> getVehicleModels() {
-        return vehicleRepository.getVehicleModels();
+    public List<VehicleModel> getVehicleModels(String fileName) {
+        return vehicleRepository.getVehicleModels(fileName);
     }
 
-    public void addAllVehicles(List<VehicleModel> listVM) {
-        vehicleRepository.addAll(listVM);
+    public void addAllVehicles(List<VehicleModel> listVM, String fileName) {
+        vehicleRepository.addAll(listVM, fileName);
     }
 
     public List<VehicleModel> findAllByInput(String input) {
-        return vehicleRepository.findAll().stream()
+        return vehicleRepository.getVehicleModels("main").stream()
                 .filter
                         (i -> i.getMark().toUpperCase().contains(input.toUpperCase())
                                 || i.getModel().toUpperCase().contains(input.toUpperCase())
                                 || i.getRegistrationNumber().toUpperCase().contains(input.toUpperCase()))
```

Comment:
It is better to extract a predicate for filtering, so it is more readable 😉 
```suggestion
                .filter(anyVehiclePropertyContains(input))
```
and predicate example:
```
private Predicate<VehicleModel> anyVehiclePropertyContains(final String input) {
        String inputUpperCase = input.toUpperCase();
        return vehicle -> vehicle.getMark().toUpperCase().contains(inputUpperCase)
                || vehicle.getModel().toUpperCase().contains(inputUpperCase)
                || vehicle.getRegistrationNumber().toUpperCase().contains(inputUpperCase);
    }
```